### ISSUE TITLE
[Reading] - Focus on Learning and Creating Rather Than Being Entertained and Distracted

### ISSUE THREAD

**Issue by viphat** [viphat/til#203]
[Source](https://medium.com/the-mission/focus-on-learning-and-creating-rather-than-being-entertained-and-distracted-e6573de1bc84)

> Ordinary people seek entertainment. Extraordinary people seek education and learning. - Benjamin Hardy.

Most people are distracted right now. They're distracted while they're at work. They're distracted when they're with family and friends. They're distracted at the gym, on their commute, and even in the shower.

The mediocre majority will continue going through life this way, never experiencing the fullness of a life filled with deep focus and purpose.

Most people don't prioritize learning and creating. They don't care enough to invest in their personal development and growth.

Entertainment is more important. Most people have replaced achieving their life dreams and goals with TV, partying, and social media.

Their life is characterized by entertainment and distraction, not learning and creating.

As a result, they don't have close relationships. They're stuck in jobs they hate. Their life is on the fast-track to disappointment, and they don't know what to do.

Entertainment and distraction is the enemy of creation and learning. They will keep you in mediocrity.

If you don't want to end up living a life of mediocrity, focus on learning and education. It's the fastest way to become extraordinary, wealthy, and successful.

**Mediocrity** (noun) = The quality of being not very good.
A goal just before half-time rescued the match from mediocrity.

### The Quality of your learning determines the quality of your success

> Your level of success will rarely exceed your level of personal development, because success is something you attract by the person become. 

Your income, relationships, and success are determined by your learning.

Most people spend more money on entertainment and gadgets than self-education. This is why they remain poor and broke with superficial relationships.

The quality of your learning and how much you invest in yourself directly determines the extent of your growth.

> The extent of the struggle determines the extent of the the growth.

You are defined by what you're willing to struggle for. Most people aren't willing to really struggle for anything. They blindly accept what they're given, and spend their free time disengaging from reality.

The most successful people in the world are always lifelong learners. They fully understand that their level of education directly determines their quality of life.

> Formal education makes a living, but self-education makes a fortune.

It's not just about a college degree. Most of what you learn in college isn't applicable to real-world success. It's your self-education, the kind of learning that helps you develop into the next revolution of yourself.

Most people will continue to go through life 'in quiet desperation, with their songs left unsung.' Others dictate their income, their happiness and their fulfillment. 

If you don't want to end up with regret on your death-bed, choose self-education over entertainment.

**Dictate** (verb) - To give orders, or tell someone exactly what they must do, with total authority.
**Desperation** - The feeling that you have when you are in such a bad situation that you are willing to take risks in order to change it.

> You've got to wake up every morning with determination if you're going to go to bed with satisfaction.

### We Attract Into Our Lives What We Are

**enormous** (extremely large)

Ask yourself - if you want a million dollars, do you have the mindset of a millionaire? Do you have the financial knowledge  and self-discipline to own that kind of money?

Most people don't, yet they continue complaining about being poor. You'll never become a millionaire if you don't become someone who could be a millionaire!

Every day, millions of people attract mediocre opportunities into their lives - jobs, time investments, dating partners, etc. - not realizing they are attracting exactly who they are.

Extremely successful people are magnets of extraordinary opportunities. They are individuals who gravitate towards success, because they are successful themselves.

If you constantly engage in mediocre, low-frequency activities with toxic individuals, you'll only receive similar opportunities. You'll never experience the type of success of the world's most extraordinary people, because **success is something you attract by the person you become**.

> **Every next level of your life will demand a different you**. - Leonardo DiCaprio

### Step forward into Growth or backwards into Security

We are like eggs at present. And you cannot go on indefinitely being just an ordinary, decent egg. We must be hatched or go bad.

You do not have the luxury to wait to improve. Like an egg, you'll either progress into a new version of yourself, or go rotten. There is no other way.

Most people continue choosing security and safety over the choice to become the fullest version of themselves.

> So, what's it going to be - safety, or freedom? You can't have both.

Most people step backward, slowly killing their chances to become extraordinary.

The sad truth is: **There's no such thing as security**. Money can be taken away, our health could fail at any time. Millions of people are abruptly fired or laid off every year.

**abrupt** (adjective) - sudden - describes something that is sudden and unexpected, and often unpleasant:
Our conversation came to an abrupt end when George burst into the room.

Security isn't real.

> The goal of life is not to relax on the beach, sipping mojitos all day. The purpose is to find something you love that adds value to the world.

### Most people will never be successful

Learning and self-education aren't popular. Most people don't read books, attend seminars, or take even the most basic steps for self-improvement. They would rather choose distraction and entertainment.

Most people will never be successful because they will spend their life this way.

With just a few lifestyle tweaks, you could enter into the top 5%-10% of your field. The competition is extremely low! Amazing, extraordinary opportunities are abundant, because so few people ever take the steps to discover them.

You can become one of the few truly successful people in the world by taking your self-education and learning seriously. In a world of mediocrity, you can become extraordinary with only a few simple tweaks.

So how do you start?

The more evolved you become, the more focused you must be on those few things which matter most.

Your days must consistently be spent on high quality activities.

Success is continuously improving who you are, how you live, how you serve, and how you relate.

Every day, check these 4 boxes: Have I improved 1% on physical, emotional, mental, and spiritual health?

You can start small. Even the smallest improvement is more than most people will ever see. While most people change slowly and unconsciously over time, you can begin taking control of your success by choosing to make a few small improvements.

That's all it takes.

### If you want to live an extraordinary life, you need to give up a normal one

Extraordinary people are very uncommon. Their lifestyle of learning and self-education aren't popular, either. Odds are, once you begin investing in yourself, you'll find it's a lonely road.

**The more bold you are, the more rejection you'll experience**.

If you want an extraordinary life, you'll need to give up many parts of the normal life. That might mean giving up common toxic behaviors or hanging out with toxic people. If everyone does it, you'd do well to give it a long, hard look.

Learning and self-education take time, energy, and focus that most people would rather spend on entertainment and distraction - 'ordinary' things.

Great opportunities cost good ones; you can't have both.

> Before you evolve, you can reasonably spend time with just about anyone. You can reasonably eat anything placed in front of you. You can reasonably justify activities and behaviors that are, frankly, mediocre.

But as your vision expands, you realize you need to make adjustments. You can't spend your money on junk anymore. You need to manage your time more diligently. You need to invest in yourself and your future.

To avoid mediocrity, you can't continue being with low-frequency, negative people.

You can't keep eating crappy foods that slow you down.

You can't do many of the things you used to do.

This is what extraordinary requires. Most people see the price and simply say 'no thanks'

But anything is possible if you pay the price.

> **Every skill you acquire doubles your odds of success.**

### In conclusion

> Who you are is a result of who you were, but where you end up depends entirely on who you choose to be from this moment forward.

There are probably dozens of recurring activities you do on a daily basis that aren't serving you.

Most people live their lives on other people's terms. Their days are spent achieving other people's goals and submitting to other people's agendas.

If you want to avoid mediocrity and achieve enormous success, you must cut out distractions and prioritize learning instead.

If your lifestyle does not add to your healing, it will subtract from it.

Prioritize your learning. Invest in self-education.

Choose success over mediocrity.
